ParisBNF/abb/BNFabb161.xml

```diff
@@ -0,0 +1,370 @@
+                          </layout>
+                        </layoutDesc>
+                      </objectDesc>
+                             
+                                 
+                      
+                      <handDesc>
+                        <handNote xml:id="h1" script="Ethiopic">
+                          <seg type="script">First half of the 19th century.</seg>
+                          <seg type="ink">Black.</seg>
+                          <desc>Careful, regular; by a good scribe (name: <persName ref="PRS14286Warqe" role="scribe">Warqe</persName>)</desc>
+                        </handNote>                       
+                     </handDesc>
+                      
+                      <additions>
+                        <list>
+                          <item xml:id="e1">
+                            <desc type="StampExlibris">
+                              Shelfmark of the Library and the title on the outer side of the front board.
+                            </desc>
+                          </item>
+                          
+                          <item xml:id="e2">
+                            <desc type="StampExlibris">Shelfmark of the Library on the spine.</desc>
+                          </item>
+                          
+                          <item xml:id="e3">                         
+                            <locus target="#ir # 1r"/>
+                            <desc type="StampExlibris"> Shelfmarks of the Library.</desc>
+                          </item>
+                        </list>
+                      </additions>
+                    </physDesc>
+                    
+                    <history>
+                      <origin>
+                      <origDate when="1848">1848</origDate>.                        
+                        <ref target="#p1">Unit 1</ref> was produced some time around or after 
+                        <date>20 Miyāzyā 1840  EC, the year of John (=27 April 1848 CE)</date> which is mentioned in 
+                        <ref target="#coloph1">Colophon 1</ref>. The scribe was <persName ref="PRS14286Warqe" role="scribe"/>.
+                      </origin>
+                    </history>
+                  </msPart>
+                  
+                  
+                      
+                      <msPart xml:id="p2">
+                  <msIdentifier/>
+                  <msContents>
+                    <summary/>
+                     <msItem xml:id="p2_i1">                      
+                        <locus from="133r" to="181r">2-94</locus>
+                       <title type="complete" ref="LIT2077Nuzhat"/>
+                        <textLang mainLang="ar"/>
+                       <colophon xml:lang="ar" xml:id="coloph2"/>                       
+                                       
+                        <note>Paginated in the reversed direction, from right to left. The colophon indicates that
```

Comment:
I would say
Paginated from right to left

Reply:
ok